shared/bundle_analysis/report.py

```diff
@@ -170,6 +171,104 @@ def ingest(self, path: str) -> int:
         self.db_session.commit()
         return session_id
 
+    def _associate_bundle_report_assets_by_name(
+        self, curr_bundle_report, prev_bundle_report
+    ) -> Set:
```

Comment:
[very nit] You can be more explicit by saying `Set[Tuple]` or even more `Set[Tuple[str, str]]`. Starts to get a bit wild fast but possible. I personally think `Set[Tuple]` is an improvement over `Set`.